packages/fast-layouts-react/src/row/row.spec.tsx

```diff
@@ -0,0 +1,568 @@
+import React from "react";
+import Adapter from "enzyme-adapter-react-16";
+import { configure, mount, shallow } from "enzyme";
+import { Row, RowClassNamesContract, RowHandledProps, RowUnhandledProps } from "./row";
+import { KeyCodes } from "@microsoft/fast-web-utilities";
+import { RowResizeDirection } from "./row.props";
+
+/**
+ * Configure Enzyme
+ */
+configure({ adapter: new Adapter() });
+
+describe("Row", (): void => {
```

Comment:
seems to me like there is a lot of duplication in row and pane, separate from this PR, should we file a task to abstract the logic into a class these can both extend and consolidate tests?

Reply:
I think that task exists: https://github.com/microsoft/fast-dna/issues/564